python/test/t_QuantileConfidence_std.py

```
@@ -460,11 +460,44 @@
+
+alpha = 0.51762756742630334
+beta = 0.95
+n = 674726183
+algo = ot.QuantileConfidence(alpha, beta)
+k1, k2 = algo.computeBilateralRank(n)
+binom = ot.Binomial(n, alpha)
+p = binom.computeCDF(k2) - binom.computeCDF(k1)
+print(f"n={n} k1={k1} k2={k2} p={p}")
+assert p >= beta
+
+for k in range(int(1e2)):
```

Ruff (0.15.15)

[warning] Loop control variable `k` not used within loop body

Rename unused `k` to `_k`

(B007)